internal/cache/profiles_test.go

```diff
@@ -158,51 +158,51 @@ func TestProfileCache_CommandExists(t *testing.T) {
 func TestProfileCache_ResourceOperations(t *testing.T) {
 	dpc := newProfileCache(dps)
 
-	if _, err := dpc.ResourceOperations(mock.NewDeviceProfile.Name, mock.NewDeviceProfile.Commands[0].Name, getOpsStr); err == nil {
+	if _, err := dpc.ResourceOperations(mock.NewDeviceProfile.Name, mock.NewDeviceProfile.Commands[0].Name, common.GetCmdMethod); err == nil {
```

Comment:
Same comment as for profiles.go applies to this file.

Reply:
will do